src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java

```diff
@@ -62,7 +60,7 @@ public void run(final UserInput userInput) {
         if (userInput.getGoal() == Goal.RELEASE) {
             reports.addAll(this.releaseUseCase.release(repository, platformNames));
         } else {
-            reports.add(this.validateUseCase.validate(repository, platformNames));
+            reports.add(this.validateUseCase.validate(repository, platformNames, Set.of(PlatformName.JIRA)));
```

Comment:
I find this a bit confusing as it's not clear why we are skipping JIRA validation. I wonder if we can express the dependency between platforms explicit and then excludes those platforms that depends on other ones.